string.c

```diff
@@ -6025,25 +6039,51 @@ str_byte_aref(VALUE str, VALUE indx)
 
 /*
  *  call-seq:
- *     str.byteslice(integer)           -> new_str or nil
- *     str.byteslice(integer, integer)   -> new_str or nil
- *     str.byteslice(range)            -> new_str or nil
+ *    byteslice(index, length = 1) -> string or nil
+ *    byteslice(range)             -> string or nil
+ *
+ *  Returns a substring of +self+, or +nil+ if the substring cannot be constructed.
+ *
+ *  With the single integer argument +index+ given,
```

Comment:
I would remove this paragraph, and move the examples following it to the next example section.  This is because behavior when passing one argument is the same as passing two arguments, with 1 as the value of the second argument.

Reply:
Done. (Duh.)